socorro/unittest/external/es/test_crashstorage.py

```diff
@@ -615,3 +612,215 @@ def esindex_fn(*args, **kwargs):
             body=document,
             **additional
         )
+
+    @mock.patch('socorro.external.es.connection_context.elasticsearch')
```

Comment:
See https://github.com/adngdb/socorro/pull/8
It requires mocking two things but it's better to go deeper so that it actually travels ConnectionContext "unmocked".